File: Solutions/462. Minimum Moves to Equal Array Elements II/462. Minimum Moves to Equal Array Elements II.go
func minMoves2(nums []int) int {
    sort.Ints(nums)
    i,j:=0,len(nums)-1
    cnt:=0
    for i<j{
        cnt+=nums[j]-nums[i]
        i,j=i+1,j-1
    }
    return cnt
}

File: Solutions/462. Minimum Moves to Equal Array Elements II/462. Minimum Moves to Equal Array Elements II.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int i=0,j=nums.size()-1;
        int cnt=0;
        while(i<j){
            cnt+=nums[j--]-nums[i++];
        }
        return cnt;
    }
};